src/util/kafka/InstanceStreamHandler.cpp

```diff
@@ -0,0 +1,91 @@
+#include "InstanceStreamHandler.h"
+#include "../../localstore/incremental/JasmineGraphIncrementalLocalStore.h"
+#include "../Utils.h"
+#include "../logger/Logger.h"
+
+Logger InstanceStreamHandlerLogger;
```

Comment:
Logger names usually defined in lower case. See for example,

`Logger data_publisher_logger;`

Reply:
Ok